features/guidance/collapse.feature

```diff
@@ -690,9 +690,9 @@ Feature: Collapse
             | restriction | bc       | dc     | c        | no_right_turn |
 
         When I route I should get
-            | waypoints | route            | turns                   | locations |
-            | a,g       | road,cross,cross | depart,turn left,arrive | a,b,g     |
-            | a,e       | road,road        | depart,arrive           | a,e       |
+            | waypoints | route                 | turns                                        | locations |
+            | a,g       | road,road,cross,cross | depart,continue slight left,turn left,arrive | a,b,c,g   |
```

Comment:
again, probably a bad interaction with collapse. It depends on which scenario is relevant here.